pages.ja/windows/find.md

```diff
@@ -0,0 +1,20 @@
+# find
+
+> 1つ以上のファイルで指定された文字列を検索します
+> 詳しくはこちら: <https://docs.microsoft.com/windows-server/administration/windows-commands/find>.
+
+- 指定された文字列を含む行を検索します:
+
+`find {{string}} {{path/to/file_or_directory}}`
+
+- 指定された文字列を含まない行を表示します:
+
+`find {{string}} {{path/to/file_or_directory}} /v`
+
+- 指定された文字列を含む行数を表示します:
+
+`find {{string}} {{path/to/file_or_directory}} /c`
+
+- 行リストとともに行番号を表示します:
```

Comment:
Did you use the same tense for this example and the previous one? just to check the output that I got from gTranslator.

Reply:
@navarroaxel 
Yes, they are using the same tense (present tense, if this is what you meant?).

Reply:
I'm getting this, but is OK :man_shrugging: 
![Screenshot_20201008_133550](https://user-images.githubusercontent.com/1536740/95487829-33320980-096b-11eb-90bc-298d6364f4f3.png)

Reply:
Ah, you are talking about the "display" and "display**s**"?
In Japanese they don't have this type of tense(I forgot what it is called), not sure why they decided to add an s for the first example.